pkg/build/api/types.go

```diff
@@ -425,6 +428,29 @@ type SourceBuildStrategy struct {
 	ForcePull bool
 }
 
+type ExternalBuilderType string
```

Comment:
so this will go away now that we're just going to explicitly declare the strategy as JenkinsPipelineBuilderStrategy, right?